canonical-web-and-design/ubuntu.com
# Remove built JS files

## Done

In [this commit](https://github.com/canonical-web-and-design/ubuntu.com/commit/96d9ba90f9fe361fe7901e8e5b2a0e64ad285806) the built JS files were added to the repository. This PR removes them from git tracking.

## QA

- Check out this feature branch
- Remove your local built JS files: `rm -rf static/js/dist`
- Run the site: `dotrun`
- Check that the build JS files have been built: `ls --ltr static/js/dist`
- Run `git status` to make sure no changes are flagged on the built JS files
